rtb/piano-di-progetto/src/pianificazione.tex

```diff
@@ -1,7 +1,578 @@
+        \item IEEE 830: Pratiche raccomandate per la specifica dei requisiti software;
+    \end{itemize}
+    \item Ricerca delle tecnologie potenzialmente adatte ad essere incluse nello
+    stack tecnologico e loro studio;
+    \item Individuazione delle tecnologie di supporto nella produzione di documenti; 
+    \item Individuazione delle tecnologie di supporto nel versionamento dei documenti e del codice prodotto
+    successivamente durante la codifica del PoC.
+\end{itemize}
+
+\graphicspath{ {./src/Gantt/} } 
+
+\begin{figure}[h] \includegraphics[scale=.6]{GanttPrimoPeriodo.png} \end{figure}
+
+\textbf{Preventivo orario}
+
+\begin{tblr}{
+    colspec={|X[5cm]|X[.5cm]|X[.5cm]|X[.5cm]|X[.5cm]|X[.5cm]|X[.5cm]|X[3.5cm]},
+    row{odd}={bg=white},
+    row{even}={bg=lightgray},
+    row{1}={bg=black,fg=white},
+    row{8}={bg=black,fg=white}
+    }
+    
+    Nominativo    & Re & Am & An & Pg & Pr & Vf & Ore Totali \\ \hline
+    Alberto C.    & -  & -  & 5  & -  & -  & 1  & 5 \\ \hline
+    Bibal El M.   & -  & -  & 4  & -  & -  & 1  & 5 \\ \hline
+    Alberto M.    & 3  & 2  & -  & -  & -  & -  & 5 \\ \hline
+    Alex S.       & 2  & 3  & -  & -  & -  & -  & 5 \\ \hline
+    Iulius S.     & -  & -  & 4  & -  & -  & 1  & 5 \\ \hline
+    Giovanni Z.   & -  & -  & 4  & -  & -  & 1  & 5 \\ \hline
+    Totale        & 5  & 5  & 17 & 0  & 0  & 4  & 30 \\ \hline
+
+\end{tblr}
+
+\textbf{Preventivo economico}
+
+\begin{tblr}{
+colspec={|X[5cm]|X[3.5cm]|X[1.5cm]|X[3.5cm]},
+row{odd}={bg=white},
+row{even}={bg=lightgray},
+row{1}={bg=black,fg=white},
+row{8}={bg=black,fg=white}
+}
+
+Ruolo & Costo orario (€/h) & N. Ore & Costo totale (€)  \\ \hline
+Responsabile      & 30,00 &  5  &   150,00 \\ \hline
+Amministratore    & 20,00 &  5  &   100,00 \\ \hline
+Analista          & 25,00 &  17 &   425,00 \\ \hline
+Progettista       & 25,00 &   0 &     0,00 \\ \hline
+Programmatore     & 15,00 &   0 &     0,00 \\ \hline
+Verificatore      & 15,00 &   4 &    60,00 \\ \hline
+Totale & \SetCell[c=1]{c} & 192 &   735,00 \\ \hline
+
+\end{tblr}
+
+\textbf{Gestione dei ruoli}
+\paragraph{}
+In questo primo periodo,il 57\% delle ore di lavoro sono state dedicate alla fase di analisi,
+data la necessità di effettuare uno studio preliminare dei documenti che dovranno essere prodotti durante
+l'arco del progetto (in particolare le Norme di Progetto e il Piano di Progetto) e quindi dei relativi Standard
+internazionali di riferimento; il 17\% delle ore è stato dedicato al ruolo del Responsabile, in cui si sono mossi i primi 
+passi nell'ambito della gestione ed organizzazione del carico di lavoro tra i vari membri; un'eguale percentuale di ore 
+è stata spesa per l'organizzazione della repository sulla piattaforma GitHub ed infine il 13\% delle ore è stato dedicato al ruolo di Verificatore,
+durante le quali i membri hanno preso dimestichezza con l'operazione di verifica e validazione dei primi documenti prodotti.
```

Comment:
In questo paragrafo fai la somma delle percentuali della divisione oraria, tuttavia la somma è: 57% (analista) + 17% (responsabile) + 17% (amministratore) + 13% (verificatore) = 104%.

Reply:
Inoltre espliciterei il fatto che l'organizzazione della repository è stata effettuata dall'amministratore.

Reply:
ok, da tonto o approssimato per eccesso tutte le percentuali